RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -1130,6 +1139,252 @@ private boolean parseSamPluginOutput(String regFilePath, AbstractFile regAbstrac
         return false;
     }
 
+    
+    /**
+     *  Finds the Host and Domain information from the registry.
+     */
+    private void parseSystemHostDomain() {
```

Comment:
We should make a new class for all of this stuff. I.e. we have a RegRipper class and our own parsing class.